packages/clerk-js/src/ui/components/PricingTable/PricingTableDefault.tsx

```diff
@@ -237,15 +242,32 @@ function Card(props: CardProps) {
               order: ctaPosition === 'top' ? -1 : undefined,
             })}
           >
-            <Button
-              elementDescriptor={descriptors.pricingTableCardActionButton}
-              block
-              textVariant={isCompact ? 'buttonSmall' : 'buttonLarge'}
-              {...buttonPropsForPlan({ plan, isCompact })}
-              onClick={event => {
-                onSelect(plan, event);
-              }}
-            />
+            <Tooltip.Root>
+              <Tooltip.Trigger
+                sx={{
+                  width: '100%',
+                }}
+                isDisabled={canManageBilling}
```

Comment:
Doesn't the button have a isDisabled prop that we check for in the tooltip to handle wrapping with span to make it focusable?

Reply:
Removed in favor of reading disabled from the child element which works from my testing. Let me know if I am missing something.